sphinx_panels/button.py

```diff
@@ -48,7 +57,7 @@ class LinkButton(SphinxDirective):
     required_arguments = 1
     final_argument_whitespace = True
     option_spec = {
-        "type": lambda arg: directives.choice(arg, ("url", "ref")),
+        "type": lambda arg: directives.choice(arg, ("url", "ref", "any", "doc", "std:ref")),
```

Comment:
This is of course not robust to list all options when going this way, I suppose (as it could also be "py:func", etc)